setup.cfg

```diff
@@ -1,4 +1,4 @@
 [tool:pytest]
-addopts = -vs --tb=short --pep8 --flakes
+addopts = -vs --tb=short --ruff --ruff-format
```

Comment:
Just noting that you're using `ruff-format` here but not in the pre-commit, which requires an extra hook `id`.